packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy.scss

```diff
@@ -18,12 +18,28 @@
     }
 
     &__container {
-        width: 77.5rem;
         transform: translateY(0);
+        @include desktop {
+            width: 77.5rem;
+        }
+        @include mobile {
+            height: calc(100% - 4rem - 4rem);
```

Comment:
or maybe we can put `4rem` under some variable, then I think it would be more descriptive such as:

```
$QUICK_STRATEGY_MARGIN: 4rem;

...
@include mobile {
    height: calc(100% - $QUICK_STRATEGY_MARGIN - $QUICK_STRATEGY_MARGIN);
    ...
}
```

do you think it would be possible and makes more sense? please let me know your view on this.

Reply:
@sandeep-deriv I think the first suggestion will be enough, thanks 👌
no sense in the second one🙂